addons/sale/static/src/js/sale_portal_prepayment.js

```diff
@@ -17,15 +17,22 @@ publicWidget.registry.PortalPrepayment = publicWidget.Widget.extend({
             'button[name="o_sale_portal_amount_prepayment_button"]'
         );
 
+        const params = new URLSearchParams(window.location.search);
```

Comment:
Moved here, because if there was not "installment amount" modal was not showing due to if (!this.AmountTotalButton).